toolsrc/include/vcpkg/base/system.process.h

```diff
@@ -34,18 +34,15 @@ namespace vcpkg::System
 
         CmdLineBuilder& path_arg(const fs::path& p) & { return string_arg(fs::u8string(p)); }
         CmdLineBuilder& string_arg(StringView s) &;
-
-        CmdLineBuilder&& path_arg(const fs::path& p) && { return std::move(path_arg(p)); }
-        CmdLineBuilder&& string_arg(StringView s) && { return std::move(string_arg(s)); };
-
-        CmdLineBuilder& ampersand() &
+        CmdLineBuilder& raw_arg(StringView s) &
         {
-            buf.push_back('&');
-            buf.push_back('&');
-            return *this;
+            buf.push_back(' ');
+            buf.append(s.begin(), s.end());
```

Comment:
```suggestion
            buf.append(s.data(), s.size());
```

Code size / compiler throughput for the data + size overloads are somewhat better because the library implementation need not reverse engineer that we have T* etc.